creator-node/nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "//": "This file is used by nodemon to watch js and ts extensions and exclude the ignore directory. This will be read by nodemon even if not explicitly passed in with --config flag",
+  "restartable": "rs",
+  "ignore": [".git", "node_modules/", "build/"],
+  "env": {
+    "NODE_ENV": "development"
```

Comment:
so does this mean nodemon only runs in dev environment?

Reply:
yes that's right. in scripts/start.sh when dev mode is false it runs straight `node`

Reply:
perf